components/x-gift-article/src/Buttons.jsx

```diff
@@ -20,19 +13,23 @@ export default ({
 	if (isGiftUrlCreated || shareType === ShareType.nonGift) {
 		if (nativeShare) {
 			return (
-				<div className={ButtonsClassName}>
-					<button className={ButtonWithGapClassNames} type="button" onClick={actions.shareByNativeShare}>
+				<div className="x-gift-article__buttons">
+					<button
+						className="js-copy-link x-gift-article__button x-gift-article-button--with-gap"
```

Comment:
question: should this be 'x-gift-article-button--gap' instead? I can't find 'x-gift-article-button--with-gap' in the components/x-gift-article/src/Buttons.scss